config/pai.service

```diff
@@ -0,0 +1,9 @@
+#sudo nano /etc/systemd/system/pai.service
+[Unit]
```

Comment:
May be helpful to add:
Wants=network-online.target
After=network-online.target